src/filtertags.md

```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  topic: [ #Logical AND
+    app, #Tasks for making apps to mobiles and tablets, such as appinventor and swift
+    electronic, #Includes external devices, such as Microbit, Arduino, Raspberry pi, lego mindstorms.
+    step_based, #Interactive web-based courses that guides you through small concepts/tasks one step at the time. Such as code.org, sessions by Khan Academy and Codecademy.
```

Comment:
vi kan godt gå for denne. "step_based" er ganske treffende, men kan også bruke "interactive". vi må også bli enige om hvordan denne oversettes.